docs/README.md

```diff
@@ -0,0 +1,46 @@
+# The Specification Document
+
+The specification document is build using [Sphinx](https://www.sphinx-doc.org/en/master/) from Markdown files through [MyST](https://myst-parser.readthedocs.io/en/latest/).
```

Comment:
```suggestion
The specification document is built using [Sphinx](https://www.sphinx-doc.org/) from Markdown files through [MyST](https://myst-parser.readthedocs.io/).
```
I think using the top level should mean the site automatically redirects based on browser locale, in case someone prefers non-en docs.